### ISSUE TITLE
Errors provisioning LLVM on Xenial (Ubuntu 16.04)

### ISSUE THREAD

**Issue by SeanCurtis-TRI** [RobotLocomotion/drake#4748]
For a sequence of nights, starting 1/5/17, two nightly builds failed to run successfully:

   - [linux-xenial-unprovisioned-gcc-nightly-release](https://drake-jenkins.csail.mit.edu/view/Nightly%20Production/job/linux-xenial-unprovisioned-gcc-nightly-release/)
   - [linux-xenial-unprovisioned-clang-nightly-release](https://drake-jenkins.csail.mit.edu/view/Nightly%20Production/job/linux-xenial-unprovisioned-clang-nightly-release/)

The problem appears to be the fact that the LLVM apt repo is not currently accessible. 

As of 1/10/17, there appeared to be no notice on the LLVM email lists, and this was confirmed as an issue independent of the CI by attempting to `apt update` on a Xenial computer.

**Comment by jamiesnape** [comment]
Possibly
https://github.com/RobotLocomotion/drake/blob/master/setup/ubuntu/16.04/install_prereqs.sh#L25
https://github.com/RobotLocomotion/drake/blob/master/setup/ubuntu/16.04/install_prereqs.sh#L31
need to be updated to
```
deb http://apt.llvm.org/xenial/  llvm-toolchain-xenial-3.9 main
```
See http://apt.llvm.org.

**Comment by jamiesnape** [comment]
Also https://github.com/RobotLocomotion/drake/blob/master/drake/doc/ubuntu_trusty.rst#c-compiler would need updating.

**Comment by jwnimmer-tri** [comment]
Yes, the new URL works.  Patch forthcoming.

**Comment by jwnimmer-tri** [comment]
Spoke too soon, not actually working yet.

**Comment by jwnimmer-tri** [comment]
Yeah, http://apt-origin.llvm.org/xenial/dists/llvm-toolchain-xenial/InRelease exists but http://apt-origin.llvm.org/xenial/dists/llvm-toolchain-xenial-3.9/InRelease is missing.  I guess LLVM is just broken?

I'm punting on this.

**Comment by wjwwood** [comment]
Should this be closed now?

**Comment by jwnimmer-tri** [comment]
Probably.  Tagging Betsy to be sure.

**Comment by BetsyMcPhail** [comment]
Yes, thank you, fixed via #4800.
